client/blocks/hello-vote-notice/index.jsx

```diff
@@ -70,6 +72,10 @@ export default connect(
 				isSiteSection( state ) &&
 				'settings' !== getSectionName( state ) &&
 				canCurrentUser( state, selectedSiteId, 'manage_options' ) &&
+				(
+					! userStartDate ||
+					moment( userStartDate ).isBefore( moment().subtract( 1, 'weeks' ) )
```

Comment:
I'm inclined to leave this as-is since my understanding is that the localized version of moment is used to display dates in the UI, and we're not doing that here.